packages/at_persistence_secondary_server/pubspec.yaml

```diff
@@ -1,12 +1,20 @@
 name: at_persistence_secondary_server
 description: A Dart library with the implementation classes for the persistence layer of the secondary server.
-version: 3.1.0
+version: 4.0.0
 repository: https://github.com/atsign-foundation/at_server
 homepage: https://docs.atsign.com/
 
 environment:
   sdk: '>=2.15.0 <4.0.0'
 
+dependency_overrides:
+  at_persistence_spec:
+    path: '../at_persistence_spec'
+  at_commons:
+    git:
+      url: https://github.com/atsign-foundation/at_libraries
+      path: packages/at_commons
+      ref: feat-immutability
```

Comment:
The above dependency overrides will be removed once those other packages have been published